Testscripts/Windows/ChangeCPU.ps1

```diff
@@ -0,0 +1,134 @@
+########################################################################
```

Comment:
Wasn't there a single script to handle cpu and/or memory changes?

Reply:
Removed duplicate testcase